### PR TITLE
Add more domains to entityComponents

### PR DESCRIPTION
**Pull request by ludeeus**
Ref: https://developers.home-assistant.io/docs/core/entity/

### CODE REVIEW

**Comment by ludeeus** [comment]
No ? <https://developers.home-assistant.io/docs/core/entity/image>

**Comment by frenck** [comment]
Yes: <https://github.com/home-assistant/core/pull/90564/>

The dev docs has an open comment on the PR, which should resolved soon.

../Frenck

File: services/bots/src/github-webhook/github-webhook.const.ts [hunk 1]
```diff
@@ -83,6 +85,7 @@ export const entityComponents = new Set([
   'siren',
   'stt',
   'switch',
+  'text',
```
Comment (frenck) [inline services/bots/src/github-webhook/github-webhook.const.ts:88]:
```suggestion
  'text',
  'time',
```

**Review by frenck — APPROVED** [review]
Thanks!

**Pull request merged**